Week3Project/Week3Project/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Week3Project
+{
+    class Program : User
+    {
+        static void Main(string[] args)
+        {
+            var user = new User();
```

Comment:
IMO, it's better to declare the types explicitly while first learning. `var` is good, but for now I would recommend putting `User user = new User();`